subsys/bluetooth/host/conn.c

```diff
@@ -53,6 +53,9 @@ struct tx_meta {
 	bool is_cont;
 };
 
+BUILD_ASSERT(sizeof(struct tx_meta) <= CONFIG_BT_L2CAP_TX_USER_DATA_SIZE,
```

Comment:
Should this be 
```suggestion
BUILD_ASSERT(sizeof(struct tx_meta) == CONFIG_BT_L2CAP_TX_USER_DATA_SIZE,
```
instead?

Reply:
yep, didn't want to be restrictive but this is more correct indeed